sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -579,15 +579,12 @@ class SparkSession private(
    *                  If a database is specified, it identifies the table/view from the database.
    *                  Otherwise, it first attempts to find a temporary view with the given name
    *                  and then match the table/view from the current database.
-   *                  Note that, the global temporary view database is also valid here.
+   *                  Note that, the global temporary view database is also valid here. Also, all
+   *                  the temporary views mentioned here include both batch and streaming plans.
```

Comment:
Let's also mention that, v2 table without batch support is not supported.

Reply:
Copy, done in c49c880